display-scene-from-mobile-scene-package/src/main/java/com/esri/arcgismaps/sample/displayscenefrommobilescenepackage/DownloadActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.esri.arcgismaps.sample.displayscenefrommobilescenepackage
```

Comment:
We could need a top-level copyright here :)

Reply:
done